docusaurus/docs/reference/ha/migrating.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Migrating
+sidebar_position: 30
+---
+
+# Migrating Controllers
+
+A controller can be moved from standalone mode to HA mode. It can also be returned
+from HA mode back to standalone mode.
```

Comment:
> from HA mode back to standalone mode

Why bother allowing people to go back? Is there a reason to support it? Also any mutation doc should start with __BACK UP YOUR CONFIG/CONTROLLER__ :)

Reply:
Added a big warning about backing up config/controller.

We allow people to go back because it doesn't require any extra code from us, and it's something someone might want to do.